### PR TITLE
Translate synchronize cache 10

### PR DESCRIPTION
**Pull request by eyhhuang**
Closes https://github.com/googleinterns/scsi2nvme/issues/131

### CODE REVIEW

File: lib/translator/translation.cc [hunk 1]
```diff
@@ -159,6 +162,10 @@ ApiStatus Translation::Complete(Span<const nvme::GenericQueueEntryCpl> cpl_data,
       pipeline_status_ = ReadToScsi(buffer, nvme_cmds_[0], kLbaSize);
       ret = ApiStatus::kSuccess;
       break;
+    case scsi::OpCode::kSync10:
```
Comment (JustinAWei) [inline lib/translator/translation.cc:165]: maybe put a todo if we need to handle something in the gen translator

File: third_party/spdk/BUILD [hunk 2]
```diff
@@ -5,5 +5,6 @@ exports_files(["LICENSE"])
 cc_library(
   name = "nvme_lib",
   hdrs = ["nvme.h"],
+  deps = ["@com_google_absl//absl/base"],
```
Comment (JustinAWei) [inline third_party/spdk/BUILD:8]: intentional?

Comment (eyhhuang) [reply third_party/spdk/BUILD:8]: yeah, nvme needs the absl dependency for the ABSL_ATTRIBUTE_PACKED annotation. It seems like before the nvme lib was always linked with the scsi lib so it somehow got the absl dependency from the scsi dependency chain? But when you try to just use the nvme lib without the scsi lib, then it breaks.

File: lib/translator/translation.cc (hunk 1 above)
Comment (eyhhuang) [reply lib/translator/translation.cc:165]: all the necessary translations should be handled by this https://github.com/googleinterns/scsi2nvme/pull/150 so once that is merged then we're all good.

File: lib/translator/BUILD [hunk 3]
```diff
@@ -19,6 +19,7 @@ cc_library(
     ":read_lib",
     ":read_capacity_10_lib",
     ":request_sense_lib",
+    ":synchronize_cache_10_lib",
```
Comment (toadchild) [inline lib/translator/BUILD:22]: Please consider naming it synchronize_cache rather than synchronize_cache_10 for consistency with other modules.

**Pull request merged**